src/commonTest/kotlin/dev/romainguy/kotlin/math/MatrixTest.kt

```diff
@@ -488,6 +488,28 @@ class MatrixTest {
         )
     }
 
+    @Test
+    fun angleFloat2() {
+        assertEquals(
```

Comment:
Is that a strict equals or does it have an epsilon? Can't be the former.

Reply:
You are right. I forgot to launch the tests on other platerforms.
![image](https://user-images.githubusercontent.com/6597529/159053283-f41829ee-61ea-41ca-839d-a3afbae4dd51.png)

Reply:
You are right. I forgot to launch the tests on other platerforms.
![image](https://user-images.githubusercontent.com/6597529/159053283-f41829ee-61ea-41ca-839d-a3afbae4dd51.png)